code/modules/hydroponics/hydroponics.dm

```diff
@@ -889,6 +889,12 @@
 	lastproduce = age
 	if(istype(myseed,/obj/item/seeds/replicapod))
 		user << "<span class='notice'>You harvest from the [myseed.plantname].</span>"
+	else if(myseed,/obj/item/seeds/spiderling

```

Comment:
This is _bad_. Replica pods aren't any better, either. Probably just make an item that the seed produces spawn a spiderlings and vanish or something.